cmd/main.go

```diff
@@ -43,23 +48,32 @@ func main() {
 
 	logger.Info("Starting: Authentication Service")
 
+	// VaultClient
+	logger.Info("Configuring Vault access 📡")
+	authenticator := coreClients.NewLocalAuthenticator("myroot")
+	vaultClient, err := coreClients.NewVaultClient(logger, vaultClientConfiguration, "authentication-service-role", authenticator)
```

Comment:
🤔  this role should be part of the configuration of the `VaultClient` in the go-core

Reply:
Let's think if we have more than one role. If the role name is moved to go-core, how the client will choose the right one?

Reply:
ok, but don't we configure this in the enviroment variable and assign the role? the client can have just one role, right? 🤔

Reply:
https://github.com/ydataai/go-core/pull/64/files

Reply:
https://github.com/ydataai/go-core/pull/64